src/new/mod.rs

```diff
@@ -98,37 +98,34 @@ fn add_dependencies(context: &mut Context) {
 }
 
 fn adjust_main_file(context: &mut Context) {
-    let mut imports = vec!["use bevy::prelude::*;"];
     let mut builder_methods = vec![];
 
     if context
         .bevy_features
         .contains(&BevyFeature::AssetHotReloading)
     {
-        imports.push("use bevy::asset::AssetServerSettings;");
-
         builder_methods.push(
             "// Enable hot reloading
-            .insert_resource(AssetServerSettings {
+            .add_plugins(DefaultPlugins.set(AssetPlugin {
                 watch_for_changes: true,
-                ..default()
-            })",
+                ..Default::default()
```

Comment:
Why was this changed, does `..default()` not work anymore?

Reply:
Purely an accidental change. I copied the code from https://bevy-cheatbook.github.io/assets/hot-reload.html and didn't notice they used `Default::default()`. Changed it back.